src/Appwrite/Migration/Version/V19.php

```diff
@@ -731,7 +730,7 @@ protected function fixDocument(Document $document): Document
 
                 if (empty($document->getAttribute('scheduleId', null))) {
                     $schedule = $this->consoleDB->createDocument('schedules', new Document([
-                        'region' => System::getEnv('_APP_REGION', 'default'), // Todo replace with projects region
+                        'region' => $project->getAttribute('region'), // Todo replace with projects region
```

Comment:
this is some old migration right? we are not going to be using this for 1.7 anyways?

Reply:
For keeping backwards compatibility functioning with new regions, its only good to replace this.